### PR TITLE
create and design work and about section for mobile devices

### PR DESCRIPTION
**Pull request by JakariaJishan**
**I design 2 sections - work and about- of my portfolio project**

1. I made sure the content of the "About myself" section is aligned with flexbox.
2. In order to lay out the elements in the "Works" section I use CSS Grid and Flexbox combined. i  Used CSS Grid in the main container that contains all of the cards and used Flexbox to lay out an individual card.
3.I used ul and li for categories in the "Work" section. Html, CSS, and Javascript.
4.I used ul and li for categories in the "About myself" section. Languages, Frameworks, and Skills.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,6 +57,257 @@ <h2>I’am a Software Developer</h2>
...
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to
+                    contact me.</p>
+
+                <button type="submit">Get My Resume</button>
+                <hr>
+            </div>
```
Comment (ridwanediallo) [inline index.html]: - [ ] Hey @JakariaJishan 👏, you did well on this project but you are missing the background image in your `about me section`. Please make sure to add it.

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @JakariaJishan , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -27,33 +27,292 @@ <h2>I’am a Software Developer</h2>
...
+                    of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to
+                    contact me.</p>
+                <button type="submit">Get My Resume</button>
+
+            </div>
+            <div class="about-img">
+                <img src="./images/about-me-img.png" alt="background img">
+            </div>
```
Comment (ichala) [inline index.html]: - [ ] for aesthetic reasons backgrounds images should be implemented using CSS background-image and not image tags, kindly fix it 🙏 [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design  ✔️ 
- No lInter errors ✔️


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (JakariaJishan) [reply index.html]: i thought i would not be the background image.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @JakariaJishan , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ichala) [reply index.html]: @JakariaJishan i didn't understand sorry can you tell me what you mean exactly?

**Pull request merged**